src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -674,20 +634,7 @@ private BoundExpression MakeImplicitConversion(BoundExpression rewrittenOperand,
                     Symbol.ReportUseSiteDiagnostic(useSiteInfo, _diagnostics, syntax.Location);
                 }
                 var fieldAccess = MakeTupleFieldAccess(syntax, field, savedTuple, null, LookupResultKind.Empty);
-
-                Conversion conversion;
-
-                HashSet<DiagnosticInfo> useSiteDiagnostics = null;
-                if (explicitCastInCode)
-                {
-                    conversion = this._compilation.Conversions.ClassifyConversionForCast(fieldAccess.Type, destElementTypes[i], ref useSiteDiagnostics);
-                }
-                else
-                {
-                    conversion = this._compilation.Conversions.ClassifyConversion(fieldAccess.Type, destElementTypes[i], ref useSiteDiagnostics);
-                }
```

Comment:
No longer need to classify conversions again at lowering (and risk to do it under different context) since we have conversions stored on the bound node.
Nullable lowering has similar changes and that is more important since in some cases we did not have all the context to redo classifications at lowering correctly.